Signal/src/ViewControllers/FingerprintViewController.m

```diff
@@ -19,26 +21,41 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface FingerprintViewController () <OWSHighlightableLabelDelegate, OWSCompareSafetyNumbersActivityDelegate>
+typedef void (^CustomLayoutBlock)();
+
+@interface CustomLayoutView : UIView
+
+@property (nonatomic) CustomLayoutBlock layoutBlock;
+
+@end
+
+#pragma mark -
+
+@implementation CustomLayoutView
+
+- (void)layoutSubviews
+{
+    self.layoutBlock();
```

Comment:
It's a little dangerous because you don't want the block to fight with the superclass layout behavior (it can cause perf problems), but I've translatesAutoresizingMaskIntoConstraints=NO so probably should be safe. Done.